compiler/plugins/target/ROCM/Dialect/ROCM/Transforms/ApplyBuiltinPDLPatterns.cpp

```diff
@@ -203,6 +204,20 @@ populatePDLModuleFromBuiltin(MLIRContext *context, RewritePatternSet &patterns,
 
 namespace {
 
+SmallVector<llvm::StringRef>
+filterUkernelPatternsByTarget(const SmallVector<llvm::StringRef> &names,
```

Comment:
```suggestion
filterUkernelPatternsByTarget(ArrayRef<StringRef> names,
```
You could use ArrayRef instead.